decidim-initiatives/app/helpers/decidim/initiatives/initiative_helper.rb

```diff
@@ -85,6 +85,26 @@ def popularity_level5?(initiative)
         initiative.percentage >= 100
       end
 
+      def authorized_create_modal_button(type, html_options, &block)
```

Comment:
Why do we have a special helper method this instead of using `action_authorized_button_to` from `Decidim::ActionAuthorizationHelper`.

There might be something I don't fully understand here...

Reply:
I followed the way it has been done with the authorized vote modal button actually.
I don't really know why it's that way then, maybe because it's connected to a special authorization handler ?